packages/contracts-bedrock/test/libraries/Encoding.t.sol

```diff
@@ -92,4 +92,120 @@ contract Encoding_Test is CommonTest {
 
         assertEq(txn, _txn);
     }
+
+    /// @dev Tests that encodeSetL1BlockValuesInterop succeeds.
+    function testFuzz_encodeSetL1BlockValuesInterop_succeeds(
+        uint32 _baseFeeScalar,
+        uint32 _blobBaseFeeScalar,
+        uint64 _sequenceNumber,
+        uint64 _timestamp,
+        uint64 _number,
+        uint256 _baseFee,
+        uint256 _blobBaseFee,
+        bytes32 _hash,
+        bytes32 _batcherHash,
+        uint256[] memory _dependencySet
+    )
+        external
+        pure
+    {
+        vm.assume(_dependencySet.length <= type(uint8).max);
+        vm.assume(uint160(uint256(_batcherHash)) == uint256(_batcherHash));
+
+        Encoding.encodeSetL1BlockValuesInterop({
```

Comment:
This test isn't super helpful given `testDiff_encodeSetL1BlockValuesInterop_succeeds` does the same thing